### PR TITLE
[DOC][MINOR] Clarification of Lerp operation on tensors

### PR DESCRIPTION
**Pull request by krishnakalyan3**
The `tensor` be used as `end` clarified in the docs. 

cc: @SsnL @fmassa @vishwakftw 

Ref 
#17205

### CODE REVIEW

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -1319,7 +1319,7 @@ def add_docstr_all(method, docstr):
                r"""
 lerp(start, end, weight) -> Tensor
```
Comment (vishwakftw) [inline torch/_tensor_docs.py]: I think this needs to be just modified as `lerp(end, weight)`.

**Review by ssnl — APPROVED** [review]
Thanks!

@pytorchbot merge this please

**Comment by krishnakalyan3** [comment]
@vishwakftw thanks for the quick review.

**Review by soumith — APPROVED** [review]

**Comment by soumith** [comment]
closed via https://github.com/pytorch/pytorch/commit/9ebc433bda3668092a4f9e1c5ecd6fcbfca5f922

**Pull request closed without merging**